tools/QuietBackgroundProcesses/DevHome.QuietBackgroundProcesses.ElevatedServer/Helpers.cpp

```diff
@@ -44,3 +50,167 @@ std::vector<ProcessPerformanceSummary> ReadPerformanceDataFromDisk(_In_ PCWSTR p
     file.close();
     return data;
 }
+
+struct ComputerInformation
+{
+    DWORD processorCount;
+    std::wstring processor;
+    std::wstring motherboard;
+    DWORDLONG ram;
```

Comment:
NIT: Include the unit size.  ramInKilobytes.  To make sure future developers keep this in kilobytes.